portfolio/src/main/java/com/google/sps/data/CommentData.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.*;
+
+/** Class containing the user comments. */
+public final class CommentData {
+
+  private final List<String> comments;
+  private final List<String> ids;
+  private int numComments;
```

Comment:
Nit: Unused variable

Reply:
These variables are to be transformed to the json string and be used from the JS script side

Reply:
Oh, I see. Could you please add a comment saying that?